msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.ObjCBinding.Common.props

```diff
@@ -17,11 +17,24 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 	<Import Project="$(MSBuildThisFileDirectory)$(MSBuildThisFileName).Before.props"
 			Condition="Exists('$(MSBuildThisFileDirectory)$(MSBuildThisFileName).Before.props')"/>
 
-	<!-- This is used to determine whether Xamarin.Mac.Common.targets needs to import 
+	<!-- This is used to determine whether Xamarin.Mac.ObjCBinding.Common.targets needs to import 
          Xamarin.Mac.ObjCBinding.Common.props itself, or whether it has been imported previously,
          e.g. by the project itself. -->
 	<PropertyGroup>
-		<_XamarinCommonPropsHasBeenImported>true</_XamarinCommonPropsHasBeenImported>
+		<_XamarinCommonBindingPropsHasBeenImported>true</_XamarinCommonBindingPropsHasBeenImported>
+	</PropertyGroup>
+
+	<Import Project="$(MSBuildThisFileDirectory)Xamarin.Mac.Common.props"
+			Condition="'$(_XamarinCommonPropsHasBeenImported)' != 'true'" />	
+
+	<PropertyGroup>
+		<BaseLibDllPath>$(MacBclPath)/Xamarin.Mac.dll</BaseLibDllPath>
+		<BTouchToolPath>$(XamarinMacFrameworkRoot)/bin/</BTouchToolPath>
+		<BTouchToolExe Condition="Exists('$(BTouchToolPath)/bgen')">bgen</BTouchToolExe>
+		<BTouchToolExe Condition="!Exists('$(BTouchToolPath)/bgen')">bmac-mobile-mono</BTouchToolExe>
```

Comment:
I'm not sure this is the right PR to do this, but `bmac-mobile-mono` should never be used, `bgen` should always be available (which means this line is redundant).

This also means we can stop installing `bmac-mobile-mono[-32]`.

Reply:
I have additional changes needed in this area:

https://github.com/xamarin/xamarin-macios/issues/3654

so I filed another for the bmac-mobile-mono bit:

https://github.com/xamarin/xamarin-macios/issues/3675